pkg/logs/status/builder.go

```diff
@@ -51,6 +55,43 @@ func (b *Builder) getIsRunning() bool {
 	return atomic.LoadInt32(b.isRunning) != 0
 }
 
+func (b *Builder) getEndpoints() []string {
+	result := make([]string, 0)
+	result = append(result, b.formatEndpoint(b.endpoints.Main, ""))
+	for _, additional := range b.endpoints.Additionals {
+		result = append(result, b.formatEndpoint(additional, "Additional: "))
+	}
+	return result
+}
+
+func (b *Builder) formatEndpoint(endpoint config.Endpoint, prefix string) string {
```

Comment:
Do we want to share this logic with the connection manager to log to which endpoint the agent connects to ?
see https://github.com/DataDog/datadog-agent/blob/master/pkg/logs/client/tcp/connection_manager.go#L60

Reply:
I'm not convinced that we should share the logic as the connection manager works at another level, i.e the TCP level so it is not aware/doesn't care of the compression and the application layer, i.e HTTP.

Reply:
sounds good